src/type-translator.ts

```diff
@@ -145,6 +148,7 @@ export class TypeTranslator {
    *     as {?}.
    * @param symbolsToPrefix a mapping from symbols (`Foo`) to a prefix they should be emitted with
    *     (`tsickle_import.Foo`).
+   * @param isAmbient whether emitting happens in an ambient context.
```

Comment:
Can you clarify this?  I think what you're saying is whether the symbols are in a module or not, but is the "ambient context" the input or the output?